### ISSUE TITLE
Assert failure: Precondition failure: FAILED: s_tracingInitialized

### ISSUE THREAD

**Issue by VincentBu** [dotnet/coreclr#25110]
**Job:**
https://mc.dot.net/#/user/coreclr-outerloop-jitstress/ci~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/test~2Ffunctional~2Fcli~2F/20190610.1/workItem/PayloadGroup0/analysis/xunit/tracing_regress._GitHub_22247_GitHub_22247_GitHub_22247_~2F_GitHub_22247_GitHub_22247_GitHub_22247_cmd

**Failed tests:**
tracing_regress._GitHub_22247_GitHub_22247_GitHub_22247_._GitHub_22247_GitHub_22247_GitHub_22247_cmd

**Log:**
```
Assert failure(PID 4704 [0x00001260], Thread: 3080 [0x0c08]): Precondition failure: FAILED: s_tracingInitialized

<no module>! <no symbol> + 0x0 (0x00000000)
    File: f:\\workspace\\_work\\1\\s\\src\\vm\\eventpipe.cpp Line: 681
    Image: C:\\dotnetbuild\\work\\10892d5f-66de-4ed2-b57f-c90665b2845d\\Payload\\CoreRun.exe


Return code:      1
Raw output file:      C:\\dotnetbuild\\work\\10892d5f-66de-4ed2-b57f-c90665b2845d\\Work\\9c9a8771-7572-4e1b-b182-f0b5fffc6d5f\\Exec\\tracing\\regress\\Reports\\tracing.regress\\GitHub_22247\\GitHub_22247\\GitHub_22247.output.txt
Raw output:
BEGIN EXECUTION
 \"C:\\dotnetbuild\\work\\10892d5f-66de-4ed2-b57f-c90665b2845d\\Payload\\corerun.exe\" GitHub_22247.exe 
Expected: 100
Actual: -1073740286
END EXECUTION - FAILED
FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=C:\\dotnetbuild\\work\\10892d5f-66de-4ed2-b57f-c90665b2845d\\Payload
> C:\\dotnetbuild\\work\\10892d5f-66de-4ed2-b57f-c90665b2845d\\Work\\9c9a8771-7572-4e1b-b182-f0b5fffc6d5f\\Exec\\tracing\\regress\\GitHub_22247\\GitHub_22247\\GitHub_22247.cmd
Expected: True
Actual:   False
```

**Comment by jorive** [comment]
I believe this ASSERTION is from my current PR: https://github.com/dotnet/coreclr/pull/25112/files. Why was this issue opened?

**Comment by BruceForstall** [comment]
@jorive Why do you think it is from your PR? The URL indicates it's from a jitstress run against master, on 6/10.

**Comment by BruceForstall** [comment]
In particular, this build: https://dev.azure.com/dnceng/public/_build/results?buildId=219698

**Comment by jorive** [comment]
@BruceForstall please ignore my comment above. Last week there was another bug opened for my previously active PR, and I did not see this one was from master. This assertion is something I'm fixing in my current PR.

**Comment by jorive** [comment]
Fixed on #25112
